OrbitQt/AccessibilityAdapter.cpp

```diff
@@ -110,7 +110,20 @@ absl::flat_hash_map<const AccessibleInterface*, QAccessibleInterface*>
 absl::flat_hash_map<const AccessibleInterface*, std::unique_ptr<AccessibilityAdapter>>
     AccessibilityAdapter::managed_adapters_;
 
+/*
+ * Utility class to verify all created accessibility adapters have been cleaned up
+ * when the application exits as a sanity check for correct registration behavior.
+ *
+ * A static instance of this is created on AccessibilityAdapter::Init() and will thus
+ * be destructed at exit. At this point, all adapters should have been cleaned up.
+ */
+class TeardownCheck {
```

Comment:
Should this be in the anonymous namespace?

Reply:
Yes, makes sense, thanks! Changed.